src/Analyzers/Core/Analyzers/PopulateSwitch/PopulateSwitchStatementHelpers.cs

```diff
@@ -71,7 +71,7 @@ public static ICollection<ISymbol> GetMissingEnumMembers(ISwitchOperation switch
             if (!TryGetAllEnumMembers(switchExpressionType, enumMembers) ||
                 !TryRemoveExistingEnumMembers(switchStatement, enumMembers))
             {
-                return SpecializedCollections.EmptyCollection<ISymbol>();
+                return [];
```

Comment:
❓ What type gets constructed for this case? I would expect this change introduced a new allocation.